quickstart.mdx

```diff
@@ -5,7 +5,12 @@ description: "How to get started with Supa"
 
 ### Create Wallet
 
-- First go to www.supa.finance. Make sure you are connected to a supported chain.
-- Go to the dashboard page and click on Create Wallet, confirm the transaction.
+- Go to www.supa.finance. Make sure you are connected to a supported chain.
+{/* Should we elaborate on the supported chains here? */}
```

Comment:
Flagging the inline comment here

Reply:
I think we should include it directly on the page since the quickstart page is usually meant to help users get started quickly by having all the info they need on one page